src/main/java/com/minecolonies/coremod/event/FMLEventHandler.java

```diff
@@ -1,13 +1,25 @@
 package com.minecolonies.coremod.event;
 
 import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.blockout.Log;
 import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.blocks.BlockConstructionTape;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.blocks.BlockConstructionTape'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)